Sources/_TestingInternals/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2024 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+include(LibraryVersion)
+add_library(_TestingInternals STATIC
+  Discovery.cpp
+  GetSymbol.cpp
+  WillThrow.cpp)
+target_include_directories(_TestingInternals PUBLIC
+  ${CMAKE_CURRENT_SOURCE_DIR}/include)
```

Comment:
One last bit I'd change is maybe passing `-fno-exceptions` to the `_TestingInternals` target build since C++ interop doesn't (at least to my knowledge) support C++ exceptions. At least from an initial scan, I think that should mean consumers of the library won't have to figure out which C++ runtime to link against. Otherwise they'll get linker errors saying that they're missing the exception handlers unless they link against a C++ runtime.